sr_utilities_common/scripts/sr_utilities_common/aws_manager.py

```diff
@@ -73,19 +74,34 @@
     )
 
     if upload_param is True:
+        uploadSucceded = False
```

Comment:
instead of waiting on params we can just import the class. But I'd set everything in 1 file with proper functions and then just call them in the main.